test/test_alexa_app_session.js

```diff
@@ -224,10 +224,9 @@ describe("Alexa", function() {
           };
         });
 
-        it("session.get(key) should not throw if attribute is not present", function() {
+        it("session.get(key) should throw if attribute is not present", function() {
```

Comment:
This now throws because we don't have a session (`{}.get(...)`). Is that correct?

```js
returnedAttributeValue = req.getSession().get("AttributeWhichDoesNotExist");
```